api/tests/Repositories/TestRepositoryTest.php

```diff
@@ -86,11 +86,14 @@ public function testCreateOrReplaceNew()
 
     public function testCreateOrReplaceUpdate()
     {
-        $entity = factory(App\Models\TestEntity::class)->create();
+        $entity = factory(App\Models\TestEntity::class)->create([
+            'varchar' => 'foobar',
+        ]);
         $id = $entity->entity_id;
 
         $entityUpdate = factory(App\Models\TestEntity::class)->make([
-            'entity_id' => $id //make sure the id doesn't change
+            'entity_id' => $id, //make sure the id doesn't change
+            'varchar' => 'foobaz',
```

Comment:
This test actually failed due to the `$faker` generating the same `'varchar'`. The dictionary must be fairly small.